### PR TITLE
Added basic GitHub usage README.

### PR DESCRIPTION
**Pull request by chrispsommers**
Adapted from https://github.com/Azure/DASH/blob/main/doc-github-rules.md which I co-authored with MSFT.
Per Slack thread: https://opi-project.slack.com/archives/C033E418VCK/p1649275592758869

Feel free to use in other dbluff/OPI repos.

### CODE REVIEW

**Review by seroyer — COMMENTED** [review]
This is a good and useful document.  However, I'm not sure it should be duplicated across all the repos.  Otherwise, if we need to make a change to it, we'd have to duplicate that change across all the repos.  Instead we could have it in the main repo and just link to it from all the others?

**Comment by chrispsommers** [comment]
@seroyer That is a good suggestion, thanks. How about if I move the "I want to Contribute" section and `doc-github-rules.md` to https://github.com/opiproject/opi` then point to those from various sub-projects, starting with `opi-poc`, let it bake a while, then follow-up with subsequent PRs to other projects?

**Comment by chrispsommers** [comment]
I submitted https://github.com/opiproject/opi/pull/3 to contain the doc and this PR was modified to link to that project. Hold off on this until the other PR is done and relative links can be tested.

File: doc-github-rules.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# GitHub basic process
+
+**Table of contents**
```
Comment (glimchb) [inline doc-github-rules.md:3]: do we have to replicate this file in each repo or can we just point to main repo https://github.com/opiproject/opi/pull/3

**Review by mestery — CHANGES_REQUESTED** [review]
Blocking until we merge https://github.com/opiproject/opi/pull/3 and adjust this to point to that one.

**Comment by seroyer** [comment]
@chrispsommers can we get this updated and closed out?  Thanks

**Comment by chrispsommers** [comment]
@seroyer I'll do it soon, I was out of town last week. Thanks for reminder.

File: doc-github-rules.md (hunk 1 above)
Comment (chrispsommers) [reply doc-github-rules.md]: Done, using common file under opi

File: README.md [hunk 2]
```diff
@@ -23,4 +23,9 @@ No specific target has been chosen, the plan was to evaluate members of [this li
 
 # Containers
 Each of the above will be running in a container.  The long term plan is to have multiple types of each container, so more than one way to generate healthy traffic, many types of malicious traffic, multiple sites to automate against, etc.
+# I Want To Contribute!
+
+This project welcomes contributions and suggestions.  We are happy to have the Community involved via submission of **Issues and Pull Requests** (with substantive content or even just fixes). We are hoping for the documents, test framework, etc. to become a community process with active engagement.  PRs can be reviewed by by any number of people, and a maintainer may accept.
+
+See [CONTRIBUTING](../opi/CONTRIBUTING.md) and [GitHub Basic Process](../opi/doc-github-rules.md) for more details.
```
[the remark this answers is not in the document]
Comment (chrispsommers) [reply README.md]: Switched to absolute links, relative didn't seem to work.

**Comment by chrispsommers** [comment]
OK, I've caught this up, sorry for the delay.

**Review by glimchb — APPROVED** [review]
lgtm

**Comment by mestery** [comment]
@chrispsommers You need to sign all commits before we can merge them, can you amend your commit with the DCO and then force push back here? Thanks!

**Pull request closed without merging**